frontend/packages/kubevirt-plugin/src/components/vms/vm-details-page.tsx

```diff
@@ -130,8 +156,12 @@ export const VirtualMachinesDetailsPage: React.FC<VirtualMachinesDetailsPageProp
       pages={pages}
       resources={resources}
       breadcrumbsFor={breadcrumbsForVMPage(props.match)}
-      customData={{ kindObj: VirtualMachineModel }}
-    />
+      customData={{ kindObj: VirtualMachineModel, updatePendingChanges }}
+    >
+      <Firehose resources={pendingChangesResources}>
```

Comment:
can we think of a better way of doing this? Without declaring the same thing twice?